README.md

```diff
@@ -1,52 +1,124 @@
+|--|-|
+| start a prompt | open your terminal or console application | open `cmd.exe` from the start menu |
+| activate the robotlab environment | type `source /path/to/robolab/bin/activate` <br/> press <kbd>ENTER</kbd> |  type `c:\path\to\robotlab\Scripts\activate` <br/> press <kbd>ENTER</kbd>
+| start robotlab | type `robotlab`<br/> press <kbd>ENTER</kbd> | type `robotlab` <br/> press <kbd>ENTER</kbd>
+
+## _Still didn't work_
+If you're still stuck, open [an issue][robotlab-issues]. There should be some
+template information to provide.
+
+## Advanced RobotLab Commands and Invocations
 ```bash
-anaconda-project run build:win
-anaconda-project run test:win
+# assuming you've already `activate`d
+robotlab [.]               # basically just jupyterlab
+robotlab-tutorial [.]      # copy the robotkernel tutorial to this directory
+robotlab-examples [.]      # "    "   "           examples "  "    "
+robotlab-shortcuts
 ```
 
-## Motivation
+## Uninstall RobotLab
+- Delete the installation directory.
+- Delete any desktop shortcuts.
+- That is all.
 
+## Motivation
 While [Robot Framework][robotframework] has no dependencies beyond the Python
 standard library, using it for non-trivial testing or process automation usually
-requires a fair amount of additional Python dependencies, and even some more
-exotic ones.
-
-The focus of this workshop, running Robot tests and tasks interactively with
-[robotkernel][], requires the Jupyter stack, and to demonstrate some of its more
-advanced features, some fairly extensive extra libraries from both the Robot
-Framework and the scientific Python ecosystems.
-
-To make a three-hour workshop reasonable, this repo uses [conda][] and
-[constructor][] to build single-file installers for multiple platforms, reducing
-the per-participant install time to a minimum, and make sneaker-net distribution
-possible in the event that network problems arise.
+requires a fair number of additional Python dependencies, and even some more
+exotic ones. We wanted to run a workshop for [RoboCon][] 2019 and show interactive
+editing/running with [robotkernel][].
+
+The focus of this workshop, running Robot tests and tasks interactively, required
+the Jupyter stack. To demonstrate some of its more advanced features, some fairly
+extensive extra libraries from both the Robot Framework and the scientific Python
+ecosystems were needed, and getting these to install smoothly on every attendee's
+computers can prove to be... interesting.
+
+To make a three-hour workshop reasonable, we needed a distribution approach that
+was:
+- easy to start, to that we could get to the Good Stuff
+- cross-platform, so that everyone could follow along
+- self-contained, so that it could be easily removed
+- offline-capable, so that we could handle intermittent network access
+- hackable by the user, so that we could fix Surprise Features
+
+> ### _Field Notes_
+>
+> _At our 2019 workshop, graciously hosted by [Silli][], (whose network was super fast!)
```

Comment:
Siili, not silli

(both are animals in Finnish, but very different :)

Reply:
104b5e0  !